spec/services/hyrax/curation_concern_spec.rb

```diff
@@ -7,4 +7,12 @@
 
     it { is_expected.to be_kind_of Hyrax::Actors::TransactionalRequest }
   end
+
+  describe ".actor_factory" do
+    subject { described_class.actor_factory }
+
+    it "returns same ActionDispatch::MiddlewareStack instance" do
+      is_expected.to eq described_class.actor_factory
```

Comment:
I wanted to test this by adding an actor the middleware stack then confirming it's still there after calling `actor_factory`, but by the time the test runs the middleware stack has been finalized and is immutable. I'm hoping just checking that actor factor returns the same instance is good enough.

Reply:
Would checking for the instance variable be a more complete test? I know it violates encapsulation…but it seems to better test what you are after.

```ruby
expect(described_class.instance_variable_get('@middleware_stack').to be_a(WhateverInstanceIsExpected™)
```

Reply:
I don't know the answer to the question, but I have no problem with making this change as well. Let me know if I should make these changes.